controllers/nnf_node_storage_controller.go

```diff
@@ -705,12 +715,36 @@ func (r *NnfNodeStorageReconciler) deleteStorage(nodeStorage *nnfv1alpha1.NnfNod
 	return &ctrl.Result{}, nil
 }
 
-func volumeGroupName(id string) string {
-	return fmt.Sprintf("%s_vg", id)
-}
+func (r *NnfNodeStorageReconciler) lvmNames(ctx context.Context, nodeStorage *nnfv1alpha1.NnfNodeStorage, id string) (string, string, error) {
+	labels := nodeStorage.GetLabels()
+
+	workflowName, ok := labels[dwsv1alpha1.WorkflowNameLabel]
+	if !ok {
+		return "", "", fmt.Errorf("Missing Workflow label on NnfNodeStorage")
```

Comment:
I know we're not consistent ourselves here; but errors strings should not be capitalized

Reply:
https://github.com/golang/go/wiki/CodeReviewComments#error-strings